source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet

```diff
@@ -1,8 +1,38 @@
 local placeholders = import '../../../lib/placeholders.libsonnet';
 local rules = import 'rules.libsonnet';
 
-local question(title) = {
-  title: title,
+local questionTitle(isProxy, isEmployed) = (
+  if isEmployed then (
+    if isProxy then
+      {
+        text: 'In <em>{person_name_possessive}</em> main job, how many hours a week do they usually work?',
+        placeholders: [
+          placeholders.personNamePossessive,
+        ],
+      }
+    else 'In your main job, how many hours a week do you usually work?'
+  ) else (
+    if isProxy then
+      {
+        text: 'In <em>{person_name_possessive}</em> main job, how many hours a week did they usually work?',
+        placeholders: [
+          placeholders.personNamePossessive,
+        ],
+      }
+    else 'In your main job, how many hours a week did you usually work?'
+  )
+);
+
+local questionDescription(isProxy, isEmployed) = (
+  if isEmployed then (
+    if isProxy then
+      ['If the <strong>coronavirus</strong> pandemic has affected their working hours, select the answer that best describes their <strong>current circumstances</strong>.']
+    else ['If the <strong>coronavirus</strong> pandemic has affected your working hours, select the answer that best describes your <strong>current circumstances</strong>.']
+  ) else []
+);
```

Comment:
I find this change hard to read, particularly the else, else. Any reason I am missing why it has been changed? I was looking for 'if the' to be added

Reply:
This is how we do such variants in jsonnet style guide, so it's up to date with the latest recommendations. I expected this change to be requested in review so I refactored in a same way @berroar did in his covid ENG/WLS content changes PR.

Reply:
Can we at least change the formatting, so the else's don't line up and it's a bit easier on the eyes

Reply:
Jsonnet format linter isn't very strict. I wonder if the style guide should be to use multiple variables to break it down to something like this which is a bit more digestible:
```jsonnet
local questionTitle(isProxy, isEmployed) = (
  local employedTitle = (
    if isProxy then
      {
        text: 'In <em>{person_name_possessive}</em> main job, how many hours a week do they usually work?',
        placeholders: [
          placeholders.personNamePossessive,
        ],
      }
    else 'In your main job, how many hours a week do you usually work?'
  );

  local unemployedTitle = (
    if isProxy then
      {
        text: 'In <em>{person_name_possessive}</em> main job, how many hours a week did they usually work?',
        placeholders: [
          placeholders.personNamePossessive,
        ],
      }
    else 'In your main job, how many hours a week did you usually work?'
  );

  if isEmployed then employedTitle else unemployedTitle
);
```
Plus isEmployed/isProxy could be flipped on our preference.

Reply:
My eyes thank you Mebin

Reply:
Ok, refactored, thanks. Do we need to update jsonnet style guide now?

Reply:
We should but prob suits a new PR with that change where all devs can review it?

Reply:
Can always reset hard and force push😉